aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -2936,6 +2947,29 @@ pub fn object_transfer_call(object: AccountAddress, to: AccountAddress) -> Trans
     ))
 }
 
+/// Create a new object to host the code, and `PublisherRef` if the code is upgradeable,
+/// Send `PublisherRef` to object signer.
+pub fn object_code_deployment_publish(
+    metadata_serialized: Vec<u8>,
+    code: Vec<Vec<u8>>,
+) -> TransactionPayload {
+    TransactionPayload::EntryFunction(EntryFunction::new(
+        ModuleId::new(
+            AccountAddress::new([
```

Comment:
There is an `AccountAddress::ONE` constant.

Reply:
This is an autogenerated function, I have a fix for this too which will be in the next PR. I'd leave this as it is.